src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -210,7 +218,10 @@ - (void)respondBusyWithDelay:(NSNumber *)delay completion:(QueryImageCompletion)
     completion(responseParams, nil);
 }
 
-- (void)respondAvailableWithDelay:(NSNumber *)delay uri:(NSString *)uri completion:(QueryImageCompletion)completion
+- (void)respondAvailableWithDelay:(NSNumber *)delay
```

Comment:
if image is available delay doesn't need to be set. is this to cover the busy use case?